lib/engines/content_block_manager/app/views/shared/_phase_banner.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render "govuk_publishing_components/components/phase_banner", {
+    phase: "Alpha",
+    message: sanitize("This is a new service - your
+        #{
+      link_to(
```

Comment:
I think you can do `mail_to`, which removes the need for the `mailto:` prefix

Reply:
ah cool, done!